lib/queuePopulator/LogReader.js

```diff
@@ -278,13 +281,22 @@ class LogReader {
     }
 
     _processLogEntry(batchState, record, entry) {
+        function _transformKey(key) {
```

Comment:
Can `_transformKey` be used anywhere else? Should it be a common util?

Reply:
I think it's too context-specific to deserve a generic function here, as backbeat relies on the version ID separator being in the key, I don't see any other possible use of this function. Also it's short so would not add much duplication if done elsewhere.